gizmo.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import numpydoc
+import pandas as pd
+
+def hello(name, country = 'Finland'):
+    print('Hello ', name, ', how are things in ', country, '?', sep = '')
+    
+def spell(word):
+    for letter in list(word)[:-1]:
+        print(letter, ".", sep="", end="")
+    print(list(word)[-1])
+    
+def relative_path(subject_identifier):
+    l = []
+    for i in range(len(subject_identifier)):
+        l += ['./subjects/mock_recording_' + subject_identifier[i] + '.rec']
+    return(l)
+
+class Gizmo:
+    def __init__(self, name):
+        self.name = name
+        
+    def speak(self):
+        print(self.name)
+
+def multiplication_table(zero_out_multiples=None):
+    """Replace all multiplies of given value with 0.
+    
+    Extended summary
+    ----------------
```

Comment:
No need to have "extended summary" as a heading. The first line will be taken as one-sentence summary, everything after that will be the extended summary.